src/rime/gear/key_binder.cc

```diff
@@ -118,7 +117,11 @@ void KeyBindings::LoadBindings(const an<ConfigList>& bindings) {
       continue;
     }
     if (auto target = map->GetValue("send")) {
-      if (!binding.target.Parse(target->str())) {
+      if (bind_sequence && !binding.target.Parse(target->str())) {
+        LOG(WARNING) << "invalid key binding #" << i << ".";
+        continue;
+      }
+      else if (!bind_sequence && !binding.target.ParseSingleKey(target->str())) {
```

Comment:
Use a `KeyEvent` local variable to parse a single key, then move to the `KeySequence`:
```c++
KeyEvent key;
if (key.Parse(...)) {
  bindings.target.push_back(std::move(key));
} else {
  // error
}
```